src/js/timeline.js

```diff
@@ -1,80 +1,49 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class Tweet extends React.Component {
+  render() {
+    let date = new Date(this.props.content.createdAt);
+    let formattedDate = date.toLocaleDateString("en-US", {month:"short", day:"2-digit"});
+    let user = this.props.content.twitterUser;
+    let content = this.props.content;
```

Comment:
if you're already pulling out the content, you should pull that out first and then reuse it vs props.content.twitterUser